lib/benchee/benchmark.ex

```diff
@@ -115,10 +131,10 @@ defmodule Benchee.Benchmark do
   end
   defp measure_runtimes(function, input, time, display_fast_warning, printer) do
     finish_time = current_time() + time
-    :erlang.garbage_collect
+    :erlang.garbage_collect()
     {n, initial_run_time} =
       determine_n_times(function, input, display_fast_warning, printer)
-    do_benchmark(finish_time, function, input, [initial_run_time], n, current_time())
+    do_benchmark(finish_time, function, input, [initial_run_time], [], n, current_time())
```

Comment:
I know it's more, but I think it'd be nice to get the initial_memory_usage along with the initial_run_time (in my mind lengt(run_times) == length(memory_usages)) - but yeah that requires some additions.